### ISSUE TITLE
Any chance of a windows version with winfsp?

### ISSUE THREAD

**Issue by fabioneves** [dweidenfeld/plexdrive#101]
Thanks to WinFSP project it might be possible?
https://github.com/billziss-gh/winfsp

Rclone seems to mount in the beta version already.

Thanks for your work!

**Comment by mitchelp** [comment]
I don't think this will be created any soon. This is just a side project until rclone does support gdrive.

**Comment by fabioneves** [comment]
I thought rclone already supported gdrive, I can even mount a gdrive in windows in the beta version.
Thanks for the input.

**Comment by savahu** [comment]
Just a correction: rclone already supports Google Drive, but usage of rclone mounts triggers Google's API limit quickly, resulting in a 24hour ban.
This is what plexdrive solves.

**Comment by xyber411** [comment]
I think this should be re-opened. Many out there that would benefit from this, especially with the newer transcoding via gpu that is finally gaining traction over at Plex.

**Comment by fabioneves** [comment]
Perhaps @billziss-gh could chime in? :)

**Comment by billziss-gh** [comment]
Hi. This can definitely be done. After all we just did it for rclone :)

OTOH I looked at the code and it uses bazil/fuse. As far as I know this presents a "VFS" like interface: the biggest difference is that file lookups are done by `(parent_dir, name)` rather than `path`. We had to modify rclone to do `path` lookups and we also changed it to use [cgofuse](https://github.com/billziss-gh/cgofuse).

So these are non-trivial changes and would require that the project maintainers/collaborators actually want to see it happen.
